services/user/Fractal/doc/fractal-service.wit

```diff
@@ -0,0 +1,78 @@
+interface types {
+    @since(feature = scheduled-meetings)
+    record meeting-config {
+        // @velua, you spent plenty of time on this for the Psinq UI. Would you fill in the minimal set of details we determined were necessary for meeting schedule?
+    }
+
+    record fractal {
+        service: account, // Primary key
+        created-at: timepointsec,
+        name: string,
+        mission: string,
+        // avatar - stored in `sites` service
+    }
+
+    
+    // Titles
+    //
+    // candidate, u32(1) << 0; 
+    // @since(feature = member-fractals)
+    // ambassador, u32(1) << 1;
+    record member {
+        account: account, // primary key
+        @since(feature = invites)
+        inviter: option<account>,
+        // display name, bio, avatar - stored in `sites` service
+        joined-at: timestamp,
+        titles: u32,
+    }
+
+    enum eval-type {
+        reputation,
+        favor,
+    }
+
+    record eval {
+        service: account,   // { Primary key,
+        id: u32,            //   Primary key }
+        type: eval-type,
+    }
+
+    record score {
+        fractal: account, // { Primary key,
+        account: account, //   Primary key,
+        type: eval-type,  //   Primary key }
+        value: u32,
+    }
+}
+
+// Don't need to pass fractal account, the caller is always the fractal
+interface fractal {
+    create-fractal: func(details: details);
+    set-name: func(name: string);
+    set-mission: func(mission: string);
+
+    create-eval: func(eval-type: eval-type) -> u32;
+    finish-eval: func(id: u32);
+}
+
+interface member {
+    join-waitlist: func(fractal: account);
+    config-candidacy: func(fractal: account, enable: bool);
+
+    @since(feature = invites)
+    invite: func(fractal: account, account: account);
+
+    @since(feature = petitions)
+    create-petition: func(fractal: account, subject: string, body: string);
```

Comment:
we'll want this returning an `id` (stg-tx-id) so callers can then reference the created petition, right?

Reply:
I don't think so, this action isn't what creates the stated tx. It will either need to accept a staged tx ID as a param, or it can just be included in the staged tx itself